examples/mlflow.ipynb

```diff
@@ -0,0 +1,749 @@
+{
```

Comment:
<p>Really nice notebook! :) </p><p> </p><p>This table seems not correctly rendered, can you pls check?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/sktime/sktime/pull/3912/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>